src/features/episode-presentation/ui/download-link.svelte

```diff
@@ -0,0 +1,17 @@
+<script lang="ts">
+  export let percentage: number | undefined = undefined;
+  export let onClickDownload: (() => void) | undefined = undefined;
```

Comment:
issue (non-blocking): that's kind of React-style to pass event handlers as props to components. 

suggestion (non-blocking): perhaps we could separate responsibilities a bit and extract the actual download link into a feature component and have that component forward the click event to the parent? The downloading progress would then be handled by another component.

The reason I think it's better is because this component's prop interface looks a bit odd to me, particularly the `percentage` prop. It's like this component implements some logic of the download, but not all of it. But I see the benefit of having it as a single component too, since it's mostly the same element in the UI.